src/main/java/by/onliner/utils/Recorder.java

```diff
@@ -0,0 +1,120 @@
+package by.onliner.utils;
+
+import org.monte.media.Format;
+import org.monte.media.Registry;
+import org.monte.media.math.Rational;
+import org.monte.screenrecorder.ScreenRecorder;
+import ws.schild.jave.*;
+import java.awt.*;
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import static org.monte.media.FormatKeys.*;
+import static org.monte.media.VideoFormatKeys.*;
+
+/***
+ * The class is used to record screen and convert video to mp4 format.
+ */
+public class Recorder extends ScreenRecorder {
+
+    public static ScreenRecorder screenRecorder;
+    private static String name;
```

Comment:
Do you need this variable to be global or you can avoid that?

Reply:
Fixed

Reply:
Comment is still actual

Reply:
Why is it static if you reinitialize it later?

Reply:
Fixed recorder